@xen-orchestra/lite/src/App.vue

```diff
@@ -38,8 +39,13 @@ const xenApiStore = useXenApiStore()
 
 const { pool } = usePoolStore().subscribe()
 
-useChartTheme()
-const uiStore = useUiStore()
+// workaround
+// since this commit https://github.com/vatesfr/xen-orchestra/commit/ac2f4e9f32beee27ce4d14ad0d4ce7d9c51a1d82
+// useUiStore is unable to find the pinia instance itself.
+const pinia = getActivePinia()
+const uiStore = useUiStore(pinia)
```

Comment:
It doesn't need to be passed to other instances of `useUiStore()`?

Reply:
No. The bug only affects this file. I don't know why. By the way, this is just a workaround. We should try to investigate and fix it more properly.